config/config_gwas_aipw.yaml

```diff
@@ -0,0 +1,17 @@
+parameters:
+    data_name: 'gwas'
+    model_name: 'aipw'
+    config_name: 'gwas_aipw'
+    ate_method_list: ['naive','aipw']
+    n_covariates: 10000
+    n_sample: 2000
+    repetitions: 25
+    seed: 4
+    max_epochs: 100
+    batch_size: 100
+    lr: 0.01
+    weight_decay: 0.05
+    use_validation: True
+    use_validation_best: True
+    use_tensorboard: False
+    alpha: [4,1,1]
```

Comment:
Excellent practice that you have .yaml files. It helps your results to be reproducible.